superflore/utils.py

```diff
@@ -59,7 +59,7 @@ def get_pkg_version(distro, pkg_name):
     return maj_min_patch
 
 
-def rand_ascii_str(length=10):
+def rand_ascii_str(length=10):  # pragma: no cover
```

Comment:
This one you could test returns only ascii characters and that the length is as expected.

Reply:
Also a good point. Thanks.